docs/system-designer/README.md

```diff
@@ -2,12 +2,13 @@
 
 The nio System Designer is the graphical user interface used to build your nio system.
 
-> Take the [nio System Designer Workshop](https://workshops.n.io/system-designer) to go hands-on with the system Designer
+{% video %}https://www.youtube.com/watch?v=GFs5EK6zao8{% endvideo %}
+
+> Take the [System Designer Workshop](https://workshops.n.io/system-designer) to go hands-on with the System Designer
```

Comment:
We don't have to fix this in this PR, but I don't really like the part that says "to go hands-on". What do you think, @jennyknuth?